packages/gestalt/src/ActivationCard.js

```diff
@@ -29,16 +30,11 @@ type LinkData = {
   target?: null | 'self' | 'blank',
 };
 
-type Props = {
+type ActivationCardProps = {
   /**
-   * Callback fired when the dismiss button is clicked (pressed and released) with a mouse or keyboard.
-   * Supply a short, descriptive label for screen-readers to provide sufficient context about the dismiss button action. IconButtons do not render text for screen readers to read requiring an accessibility label.
-   * Accessibility: `accessibilityLabel` populates aria-label.
+   * Adds a dismiss button to BannerOverlay. See the [Dismissible variant](https://gestalt.pinterest.systems/web/activationcard#Dismissible) for more info.
```

Comment:
typo
 Adds a dismiss button to **BannerOverlay**. See the [Dismissible variant](https://gestalt.pinterest.systems/web/activationcard#Dismissible) for more info.

Reply:
Sorry this was part of outdated commit because I hadn't turned on vscode autosave yet